toolchain/lower/testdata/interop/cpp/function_decl.carbon

```diff
@@ -112,7 +112,6 @@ fn MyF() {
 // CHECK:STDOUT: target triple = "x86_64-unknown-linux-gnu"
 // CHECK:STDOUT:
 // CHECK:STDOUT: define void @_CMyF.Main() !dbg !7 {
-// CHECK:STDOUT: entry:
```

Comment:
Not sure why `entry:` got removed here (because of `-fno-exceptions` maybe?). It also doesn't really matter at this point, probably.